bin/p2-log-bridge/main.go

```diff
@@ -20,31 +21,57 @@ func main() {
 	kingpin.Version(version.VERSION)
 	kingpin.Parse()
 
+	if err := setSTDINToBlock(); err != nil {
+		logging.DefaultLogger.WithError(err).Error("fatal error setting STDIN to block")
+		os.Exit(1)
+	}
+
+	var wg sync.WaitGroup
+	fmt.Printf("durableLogger: %v", *durableLogger)
 	loggerCmd := exec.Command(*durableLogger)
 	durablePipe, err := loggerCmd.StdinPipe()
 	if err != nil {
-		logging.DefaultLogger.WithError(err).Errorln("failed during logbridge setup")
-		os.Exit(1)
+		logging.DefaultLogger.WithError(err).Error("fatal error during configuration of subordinate log command")
 	}
 
-	go handleSignals(durablePipe)
+	go func(loggerCmd exec.Cmd, wg sync.WaitGroup) {
+		wg.Add(1)
+		defer wg.Done()
 
-	go func(r io.Reader, durableWriter io.Writer, nonDurableWriter io.Writer, logger logging.Logger) {
-		logbridge.Tee(r, durableWriter, nonDurableWriter, logger)
-	}(os.Stdin, durablePipe, os.Stdout, logging.DefaultLogger)
+		if err := loggerCmd.Start(); err != nil {
+			logging.DefaultLogger.WithError(err).Errorln("fatal error during execution of subordinate log command")
+			os.Exit(1)
+		}
 
-	loggerCmd.Start()
-	err = loggerCmd.Wait()
-	if err != nil {
-		logging.DefaultLogger.WithError(err)
-		os.Exit(1)
-	}
+		err := loggerCmd.Wait()
+		if err != nil {
+			logging.DefaultLogger.WithError(err)
+			os.Exit(1)
+		}
+	}(*loggerCmd, wg)
+
+	logbridge.Tee(os.Stdin, durablePipe, os.Stdout, logging.DefaultLogger)
+	logging.DefaultLogger.NoFields().Info("logbridge Tee returned. Shutting down subordinate log command.\n")
+	durablePipe.Close()
+
+	logging.DefaultLogger.NoFields().Info("logging running in background…")
```

Comment:
Typographical correctness aside, I think it's safer to avoid the whole issue of non-ASCII output and prefer `...` to `…`.